.github/workflows/sycl-linux-precommit.yml

```diff
@@ -60,7 +60,7 @@ jobs:
         id: arc_tests
         run: |
           if [ "${{ contains(needs.detect_changes.outputs.filters, 'devigccfg') }}" == "true" ]; then
-            echo 'arc_tests="(ESIMD|InvokeSimd|Matrix)/"' >> "$GITHUB_OUTPUT"
+            echo 'arc_tests="(ESIMD|InvokeSimd|Matrix|bindless_images)/"' >> "$GITHUB_OUTPUT"
```

Comment:
Why do we need this.

Reply:
Is the filter here decides which e2e tests are running in pre-CI?
I see only matrix tests are running in pre-CI.
My intention is to enable bindless_image in the CI. Are bindless_image tests running in postCI?

Reply:
Are they requiring Arc GPUs? How much more resources would these tests take? We only run full testing on this class in post-commit because of the limited HW resources in CI.

Also, this line is responsible for testing driver updates, not regular pre-commit CI for the changes in the SYCL.

Reply:
Yes, they requires DG2. The tests enabled in this PR take 60s on my local DG2 ubuntu machine.

> Also, this line is responsible for testing driver updates, not regular pre-commit CI for the changes in the SYCL.

I see, thank you.

Do you know what is the intel gpu driver version that is used for post commit task "Intel Arc A-Series Graphics with Level Zero"? This PR requires a recent gpu driver.

Reply:
Changes to this file is reverted.

Reply:
> Do you know what is the intel gpu driver version that is used for post commit task "Intel Arc A-Series Graphics with Level Zero"? This PR requires a recent gpu driver.

I think it's https://github.com/intel/llvm/blob/sycl/devops/dependencies.json, @jsji or @sarnex might correct me if I'm wrong.

Reply:
thank you. Even the latest igc release [igc-1.0.16695.4](https://github.com/intel/intel-graphics-compiler/releases/tag/igc-1.0.16695.4) is still missing a patch for bindless image support. So the tests in this PR will fail in post-commit testing.

Is there a way to use https://github.com/intel/llvm/blob/sycl/devops/dependencies-igc-dev.json for task "Intel Arc A-Series Graphics with Level Zero" in post-commit testing?

Reply:
It looks like we always use the released driver in postcommit.

Reply:
> Even the latest igc release [igc-1.0.16695.4](https://github.com/intel/intel-graphics-compiler/releases/tag/igc-1.0.16695.4) is still missing a patch for bindless image support.

I was wrong about IGC release. The missing patch is for windows, but we don't run these tests on windows.

Unsampled tests should already be passing with released drivers in https://github.com/intel/llvm/blob/sycl/devops/dependencies.json

Sampled tests depends on https://github.com/intel/compute-runtime/commit/76af952fcbcea9501ebc12c9d2eff0bc4b63da63 which is not yet in [Latest NEO release](https://github.com/intel/compute-runtime/releases/tag/24.17.29377.6)